reframe/frontend/executors/policies.py

```diff
@@ -424,32 +458,53 @@ def runcase(self, case):
     def _poll_tasks(self):
         '''Update the counts of running checks per partition.'''
 
-        def split_jobs(tasks):
+        def split_jobs(tasks, build_split=False, build_jobs=False):
```

Comment:
Why do you need to differentiate the build jobs here?

Reply:
I am not sure why I put two extra arguments here, I think just by mistake. I will keep one since `split_jobs` needs to know if it needs to return a list of build_jobs or the run jobs.